hack/chart-verifier-docker.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# KUBECONFIG is either the value the user has configured in the
+# environment or the default location `kubectl` uses.
+KUBECONFIG="${KUBECONFIG:-${HOME}/.kube/config}"
+
+# CHART_VERIFIER_IMAGE is the location of the image to use to verify
+# the chart.
+CHART_VERIFIER_IMAGE="${CHART_VERIFIER_IMAGE:-quay.io/redhat-certification/chart-verifier:main}"
+
+# DOCKER contains the path for the `podman` program.
+DOCKER="${DOCKER:-/usr/bin/docker}"
+
+"${DOCKER}" run --rm -i                      \
```

Comment:
with the detection above we can collapse these two scripts into single one:

```
$CONTAINER_RUNTIME run --rm -e KUBECONFIG=/.kube/config \
  -v "${KUBECONFIG}":/.kube/config \
  "${CHART_VERIFIER_IMAGE}"  "$@"
```

Reply:
Since we have just single cmd `verify`, we can even append it by default:

```
$CONTAINER_RUNTIME run --rm -e KUBECONFIG=/.kube/config \
  -v "${KUBECONFIG}":/.kube/config \
  "${CHART_VERIFIER_IMAGE}"  verify "$@"
```

so that the script can be called like `chart-verifier.sh https://foo/chart.tgz`

Reply:
Is this supposed to be only a shim to support automated tooling or trying to enable some features but using the same semantics?

The approach I've taken was to provide the same semantics the program has, instead of simplifying like this.